### PR TITLE
UX: Improve style of GitHub PR body in emails

### PR DESCRIPTION
**Pull request by udan11**
On web, we display only an excerpt in a monospace font and the email did
not use the same style.

image

### CODE REVIEW

File: lib/email/styles.rb [hunk 1]
```diff
@@ -139,6 +139,8 @@ def onebox_styles
       style('.github-info div', "display: inline; margin-right: 10px;")
       style('.github-icon-container', 'float: left;')
       style('.github-icon-container *', 'fill: #646464; width: 40px; height: 40px;')
+      style('.github-body-container', 'font-family: Consolas, Menlo, Monaco, "Lucida Console", "Liberation Mono", "DejaVu Sans Mono", "Bitstream Vera Sans Mono", "Courier New", monospace; margin-top: 1em !important;')
```
Comment (eviltrout) [inline lib/email/styles.rb:142]: Do we need the `!important` there? It should only be used as a last case resort.

File: lib/email/styles.rb [hunk 2]
```diff
@@ -139,6 +139,8 @@ def onebox_styles
...
+      style('.github-body-container .excerpt', 'display: none;')
```
Comment (ZogStriP) [inline lib/email/styles.rb:143]: Can we trim the excerpt instead of hiding it?

File: lib/email/styles.rb (hunk 1 above)
Comment (udan11) [reply lib/email/styles.rb:142]: In `correct_first_body_margin` from below, `margin-top:0` is set for all `<p>` elements. I could not target the parent element either, so I had to use `!important`.

File: lib/email/styles.rb [hunk 3]
```diff
@@ -140,9 +140,12 @@ def onebox_styles
       style('.github-icon-container', 'float: left;')
       style('.github-icon-container *', 'fill: #646464; width: 40px; height: 40px;')
       style('.github-body-container', 'font-family: Consolas, Menlo, Monaco, "Lucida Console", "Liberation Mono", "DejaVu Sans Mono", "Bitstream Vera Sans Mono", "Courier New", monospace; margin-top: 1em !important;')
-      style('.github-body-container .excerpt', 'display: none;')
       style('.onebox-avatar-inline', ONEBOX_INLINE_AVATAR_STYLE)
 
+      @fragment.css('.github-body-container .excerpt').each do |el|
+        el.remove
+      end
+
```
Comment (ZogStriP) [inline lib/email/styles.rb:148]: I think you can call `remove` (or `delete`) directly

```suggestion
      @fragment.css('.github-body-container .excerpt').remove
```

Comment (udan11) [reply lib/email/styles.rb]: Nice.

**Review by ZogStriP — APPROVED** [review]

**Comment by ZogStriP** [comment]
Maybe add a test as well to ensure we don't regress?

**Pull request merged**